packages/website/docs/_samples/ai/Input/Basic/sample.html

```diff
@@ -0,0 +1,21 @@
+<!-- playground-fold -->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sample</title>
+</head>
+
+<body style="background-color: var(--sapBackgroundColor)">
+<!-- playground-fold-end -->
```

Comment:
wrap the ui5-ai-input in a div for the actions menu to be visible in the documentation sample:
`<div style="height:150px">`